### PR TITLE
Minor additions and fixes to README

### PR DESCRIPTION
**Pull request by louieyoung98**
WIP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,17 +14,19 @@ And then execute:
 
     $ bundle install
 
+**Reminder:** If you have a rails server running, remember to restart the server to prevent the `uninitialized constant` error.
+
 ## Usage
 
 This gem provides two entry points: Frontend for use in your views, and Backend for use in your controllers. The backend
-is option, it's only required if you want to support sorting by column headers.
+entry point is opitonal, as it's only required if you want to support sorting by column headers.
```
Comment (sfnelson) [inline README.md:22]: check your spelling :-D

File: README.md [hunk 2]
```diff
@@ -14,17 +14,19 @@ And then execute:
...
 
 ### Frontend
 
 Add `include Katalyst::Tables::Frontend` to your `ApplicationHelper` or similar.
 
 ```erb
-<%= table_with collection: @people do |row, person|
+<%= table_with collection: @people do |row, person| %>
```
Comment (sfnelson) [inline README.md:29]: 👍

**Review by sfnelson — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (louieyoung98) [reply README.md]: Hah whoops, thanks!

**Review by sfnelson — APPROVED** [review]
Thanks for fixing this up Louie!

**Pull request merged**